docker/experimental/ansible/config/apt.yaml

```diff
@@ -0,0 +1,63 @@
+# Contains lists of apt packages for each stage (build|release) and arch or accelerator.
+apt:
+  pkgs:
+    build_common:
+      - ccache
+      - curl
+      - git
+      - gnupg
+      - libomp5
+      - libopenblas-dev
+      - ninja-build
+      - procps
+      - python3-pip
+      - rename
+      - vim
+      - wget
+
+    build_cuda:
+      - cuda-libraries-11-2
```

Comment:
Recently [bumped](https://github.com/pytorch/xla/commit/a497db46e7fb7b58be7584aed69ac7477428b3c6) to 11.7, but in the circleci, not in docker. Hm.

@JackCaoG what's the source of truth here?

Reply:
@yeounoh

Reply:
Right now, the builds are very fragmented, so the release images haven't been updated. If possible, we should use 11.7 or 11.8 going forward. If the change here isn't trivial, then we can push the work to a follow up PR

Reply:
Done